bin/installer

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env php
```

Comment:
This is copied (almost) verbatim from Composer. Since I'm running Composer from within I thought it might be prudent to use its setup.